.lycheeignore

```diff
@@ -0,0 +1,40 @@
+^getting-started/
+^production-environment/
+^product-tour/
+^demos/
+^architecture/
+^operations/
+^upgrading/
+^README.md
+^multi-tentant-environment.md
+^openshift-environment.md
+^help-and-support/
+^https?://localhost
+^https://mvnrepository.com
+^https://platform.openai.com/docs
+^http://default-broker.seldon-logs
+^https://www.tensorflow.org/tfx/serving/api_rest#request_format_2
+^https://otexts.org/fpp2/holt-winters.html
+^https://www.pcmag.com/encyclopedia
+^https://www.microsoft.com/en-us/download/details.aspx?
+^https://opensea.io/
+^https://www.techopedia.com/definition/31996/hash-partitioning
+^https://docs.github.com/
+_template
+^https://127\.0\.0\.1(:\d+)?/
```

Comment:
may be you can add the http:// as well. It was reported https://github.com/SeldonIO/seldon-core/actions/runs/16800884316?pr=6664

<img width="514" height="59" alt="Image" src="https://github.com/user-attachments/assets/94d93f68-067b-443d-9ecc-789ddb116cfc" />

Reply:
> @Rajakavitha1 did you mean here `http://127.0.0.1` or just `http://`?
> 
> Monica added `^http://`, but I don't think it's valid to ignore **all links starting from http://** as there could be links we would like to be really working
> 
> cc @monica-seldon

Hi @monica-seldon  and @tyndria  could you please ensure that http://127.0.0.1 is not reported as broken link.

Thank you.